packages/webview_flutter/example/ios/Runner.xcodeproj/xcshareddata/xcschemes/RunnerUITests.xcscheme

```diff
@@ -1,52 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This scheme is unnecessary, the CI tests use the test action of the `Runner` scheme (which runs the UI and unit tests).